data/bactrian.txt

```diff
@@ -81,10 +81,10 @@ mission "Bactrian Unlock [0]"
 
 	on enter Arneb
 		set "found star queen"
-		dialog "As you enter Arneb, you spot the Star Queen landing on Haven, defended by a number of pirate vessels. You will need to report back to the Deep in order to retrieve the Star Queen in one piece."
+		dialog `As you enter Arneb, your scanners pick up the Star Queen landing on Haven. Unfortunately, there are also a large number of pirate vessels headed in your direction.  You should report back to the Deep Security officer on <destination> for further instruction.`
```

Comment:
this probably depends on your dialect or something, but also consider changing "instruction" to "instructions".